mobile/src/api/client.ts

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+import {RegisterRequest, SignInRequest, SignInResponse} from './types';
+const baseUrl = 'http://10.0.2.2:3000/v1';
+
+export const authApi = axios.create({
+  baseURL: baseUrl,
+  timeout: 10000,
```

Comment:
To bym wrzucił do jakiegoś contsta może żeby tym sterować z jednego miejsca

Reply:
masz na myśli timeout?